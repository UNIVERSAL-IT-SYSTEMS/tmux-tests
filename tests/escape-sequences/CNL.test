#!/bin/sh

S=$(mktemp)
trap 'rm -f $S' 0 1 15

cat <<EOF >$S
stty tabs

printf "\033[Ea"
printf "\tb\033[Ec"
printf "\td\033[2Ee"
printf "\033[24df\033[Eg"

$TMUX_L wait -S tests
$TMUX_L wait never
EOF

$TMUX_L new -d -x 80 -y 25 -- sh $S
$TMUX_L wait tests\; capturep -ep -S- -E-
