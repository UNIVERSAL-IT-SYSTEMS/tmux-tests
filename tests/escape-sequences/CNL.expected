
a       b
c       d

e


















gf
